doc/book/getting-started/forms-and-actions.md

```diff
@@ -102,15 +102,6 @@ class Album implements InputFilterAwareInterface
         $this->title  = ! empty($data['title']) ? $data['title'] : null;
     }
 
-    public function getArrayCopy()
```

Comment:
This isn't duplicated, though! Nowhere else in the class definition presented here is the method defined...